chain/network/src/peer/peer_actor.rs

```diff
@@ -565,6 +569,20 @@ impl PeerActor {
             }
         }
 
+        // Verify the sender's peer_id has signed on the stream's ip_addr for both inbound and outbound connection
```

Comment:
See other comments in this function; simplify this to something like "Verify the signed IP Address is valid."

Reply:
Updated as suggested to  // Verify the signed IP address is valid.